cmake/Fluid.cmake

```diff
@@ -18,7 +18,7 @@ else()
   set(USE_VALENCIA 0 CACHE BOOL "Covariant formulation of GRMHD")
 endif()
 
-option(PHOEBUS_C2P_ROBUST_FLOORS "Use full floors for c2p robust" OFF)
+option(PHOEBUS_C2P_ROBUST_FLOORS "Use full floors for c2p robust" ON)
```

Comment:
Do we want this `ON` by default? My understanding from your recent disk runs is that for torus calculations we will always want this `OFF`

Reply:
That's not the sense I have.  My sense was actually the opposite.  It seems to actually work quite well with it `ON`.  We clearly need to do some more quantitative analysis as you suggested offline, but I'd vote for this to be `ON` by default, for now at least.

Reply:
Oh OK I thought with the full floors that we were getting a big halo around the torus initially. If that's not true that's great.